<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Levels</class>
 <widget class="QWidget" name="Levels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="buttonSwitch">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Edit Level</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNew">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>New Level</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDelete">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Delete Level</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Level Mapping</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>20</number>
        </property>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="spacing">
           <number>12</number>
          </property>
          <item row="3" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label5b">
            <property name="text">
             <string>5b</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label5c">
            <property name="text">
             <string>5c</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio5b">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio5c">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label5a">
            <property name="text">
             <string>5a</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio5d">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio5a">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio6a">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio6c">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio6b">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio6d">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="4" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label5e">
            <property name="text">
             <string>5e</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label5d">
            <property name="text">
             <string>5d</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio5e">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="4" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio6e">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label6a">
            <property name="text">
             <string>e1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label6b">
            <property name="text">
             <string>e2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label6c">
            <property name="text">
             <string>e3</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label6d">
            <property name="text">
             <string>e4</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label6e">
            <property name="text">
             <string>e5</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="verticalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="0" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio1">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="label1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="spacing">
           <number>12</number>
          </property>
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio4c">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3" alignment="Qt::AlignLeft">
           <widget class="QRadioButton" name="radio4d">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio4b">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignRight">
           <widget class="QRadioButton" name="radio4a">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label4a">
            <property name="text">
             <string>4a</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label4b">
            <property name="text">
             <string>4b</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label4c">
            <property name="text">
             <string>4c</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label4d">
            <property name="text">
             <string>4d</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="horizontalSpacing">
           <number>45</number>
          </property>
          <property name="verticalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="0" alignment="Qt::AlignRight">
           <widget class="QRadioButton" name="radio2a">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radio2b">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QLabel" name="label2a">
            <property name="text">
             <string>2a</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QLabel" name="label2b">
            <property name="text">
             <string>2b</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="spacing">
           <number>12</number>
          </property>
          <item row="0" column="2" alignment="Qt::AlignLeft">
           <widget class="QRadioButton" name="radio3c">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radio3b">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignRight">
           <widget class="QRadioButton" name="radio3a">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label3a">
            <property name="text">
             <string>3a</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label3b">
            <property name="text">
             <string>3b</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label3c">
            <property name="text">
             <string>3c</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="checkStartLine">
          <property name="text">
           <string>Display Start Line</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="buttonMap">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Map Level</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
